clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -108,13 +105,13 @@
 import static org.apache.kafka.common.utils.Utils.propsToMap;
 
 /**
- * This prototype consumer uses an {@link ApplicationEventHandler event handler} to process
- * {@link ApplicationEvent application events} so that the network IO can be processed in a dedicated
+ * This {@link Consumer} implementation uses an {@link ApplicationEventHandler event handler} to process
```

Comment:
Similarly - Can we add a description stating the module is used to support KIP-848 protocol?

Reply:
Will do.

Reply:
Done. If you want the verbiage changed in some fashion, just let me know. Thanks!